exercises/G-numbers/exercise.js

```diff
@@ -1 +1,11 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15
+var numberOfMentors = 8
```

Comment:
Great work Rafael, I would only make a  small comment around your use of semi-colons that are missing on a few lines.